rott/cin_actr.c

```diff
@@ -24,7 +24,6 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 #include "cin_efct.h"
 #include "modexlib.h"
 //MED
-#include "memcheck.h"
```

Comment:
I guess the `//MED` comment wants to leave us together with the `#include "memcheck.h"` line.

Reply:
I've learned to ignore any comments in ROTT's source code. My bad!